nbgitpuller/plugin_helper.py

```diff
@@ -205,14 +203,14 @@ async def handle_files_helper(helper_args, query_line_args):
     provider = query_line_args["content_provider"]
     repo_parent_dir = helper_args["repo_parent_dir"]
     origin_repo = f"{repo_parent_dir}{CACHED_ORIGIN_NON_GIT_REPO}{provider}/{url}/"
-    temp_download_repo = TEMP_DOWNLOAD_REPO_DIR
+    temp_download_dir = tempfile.TemporaryDirectory(dir="/tmp")
```

Comment:
Do we have a need to have a root location specified? Otherwise i suggest we remove this /tmp reference as it is unix specific.

Reply:
yes got it.